### ISSUE TITLE
Sparse label training with very low ACC

### ISSUE THREAD

**Issue by SC-shendazt** [HuguesTHOMAS/KPConv-PyTorch#211]
hello, Dr.HuguesTHOMA,  
![QQ图片20230305214019](https://user-images.githubusercontent.com/74888508/222964029-7c6e7328-b19e-4ae9-b023-00ebbde51d7a.png)
![QQ图片20230305214111](https://user-images.githubusercontent.com/74888508/222964034-30f9b433-a570-488f-ad78-7838a085c817.png)
When I modified the loda_subsampled_clouds function to facilitate label sparse extraction for sub_lables, and the procedure runs, it is very strange that the ACC is almost 0, but there are results when validating. As shown below, could you give me some advice?

**Comment by HuguesTHOMAS** [comment]
In this case, I won't be able to help too much, because you are trying new stuff that was not in my original code. You have to debug what happens, try printing the values of the predicted labels, are they all zeros? Try verifying the target labels in the loss function of the network, etc...
